unit-tests/src/test/scala/scala/scalanative/regex/ExecTest.scala

```diff
@@ -0,0 +1,655 @@
+
+// import java.io.InputStreamReader
+// import java.io.File
+// import java.io.FileInputStream
+// import java.{util => ju}
+// import java.util.regex.PatternSyntaxException
+// import java.util.zip.GZIPInputStream
+// import java.util.{Arrays, Collections}
+
+// import scala.util.control.NonFatal
+// import scala.util.control.Breaks._
+
+import org.junit.Ignore
+import org.junit.Test
+import org.junit.Assert._
+
+// TestRE2 tests this package's regexp API against test cases
+// considered during (C++) RE2's exhaustive tests, which run all possible
+// regexps over a given set of atoms and operators, up to a given
+// complexity, over all possible strings over a given alphabet,
+// up to a given size.  Rather than try to link with RE2, we read a
+// log file containing the test cases and the expected matches.
+// The log file, re2.txt, is generated by running 'make exhaustive-log'
+// in the open source RE2 distribution.  http://code.google.com/p/re2/
+//
+// The test file format is a sequence of stanzas like:
+//
+//      strings
+//      "abc"
+//      "123x"
+//      regexps
+//      "[a-z]+"
+//      0-3;0-3
+//      -;-
+//      "([0-9])([0-9])([0-9])"
+//      -;-
+//      -;0-3 0-1 1-2 2-3
+//
+// The stanza begins by defining a set of strings, quoted
+// using Go double-quote syntax, one per line.  Then the
+// regexps section gives a sequence of regexps to run on
+// the strings.  In the block that follows a regexp, each line
+// gives the semicolon-separated match results of running
+// the regexp on the corresponding string.
+// Each match result is either a single -, meaning no match, or a
+// space-separated sequence of pairs giving the match and
+// submatch indices.  An unmatched subexpression formats
+// its pair as a single - (not illustrated above).  For now
+// each regexp run produces two match results, one for a
+// ``full match'' that restricts the regexp to matching the entire
+// string or nothing, and one for a ``partial match'' that gives
+// the leftmost first match found in the string.
+//
+// Lines beginning with # are comments.  Lines beginning with
+// a capital letter are test names printed during RE2's test suite
+// and are echoed into t but otherwise ignored.
+//
+// At time of writing, re2.txt is 32 MB but compresses to 760 kB,
+// so we store re2.txt.gz in the repository and decompress it on the fly.
+
+@Ignore("takes too long to run")
+class ExecTest {
+
+  /* The following dummy test was added during the JUnit
+   * port and the rest of the file is commented out because
+   * of compiler errors. Having one test means the @Ignore
+   * on the test works and will show output when run as
+   * part of the rest of the tests.
+   *
+   *
+   * When the rest of the file is uncommented the tests
+   * ran despite of the compile errors but slowly as noted above.
+   */
+  @Test def dummyTestBecauseFileWontCompileClean(): Unit = {
+    assertTrue(true)
+  }
+
+  // @Test def examplesInDocumentation(): Unit = {
```

Comment:
Please use `/* */` around all the tests to comment them out, instead of using a `// ` prefix. That way, if we manage to enable them at some point, we won't have `git diff` telling us that all the lines have changed.

(Also it might even make git understand that this file is a rename of `ExecTest.scala.disabled`.

Reply:
Really good point, thanks.